### ISSUE TITLE
hitting: 'pulumi new' error: cloning templates repo: reference not found

### ISSUE THREAD

**Issue by nomiali5** [pulumi/pulumi#12277]
### What happened?

After installing pulumi, creating a new pulumi project following **Get started with pulumi** , Am I missing something here? Tried excuting **pulumi new gcp-csharp** but facing same error.

### Expected Behavior

It should show the templates to choose from.

### Steps to reproduce

choco install pulumi
pulumi new

### Output of `pulumi about`

Pulumi locates its logs in C:\Users\nomiali\AppData\Local\Temp by default
warning: Failed to read project: no Pulumi.yaml project file found (searching upwards from C:\Users\nomiali\Desktop\Learning Pulumi\Demo2). If you have not created a project yet, use `pulumi new` to do so: no project file found
warning: Failed to get information about the current stack: no Pulumi.yaml project file found (searching upwards from C:\Users\nomiali\Desktop\Learning Pulumi\Demo2). If you have not created a project yet, use `pulumi new` to do so: no project file found

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by justinvp** [comment]
Hi @nomiali5, sorry for the trouble. I haven't been able to reproduce the error you're seeing on macOS, but someone will try to reproduce on Windows.

In the meantime, I'm curious, do you have a `C:\Users\nomiali\.pulumi\templates` directory? Try deleting the `templates` directory and re-run `pulumi new gcp-csharp` again.

**Comment by nomiali5** [comment]
Yes, after deleting the `C:\Users\nomiali\.pulumi\templates` directory, now I am able to see the templates selection and able to run `pulumi new gcp-csharp`. Thanks
